tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -98,7 +126,162 @@ module('Acceptance | guid file | registration files', hooks => {
             .hasAria('label', t('file_detail.view_revisions'), 'Versions button has correct label when closed');
     });
 
-    // test for tags
+    // Test file metadata
+    test('view metadata', async function(this: ThisTestContext, assert) {
+        const nodeNoun = ['Registration','Project','Component'];
+        setBreakpoint('desktop');
+        await visit(`/--file/${this.file.id}`);
+        assert.equal(currentURL(), `/--file/${this.file.guid}`);
+
+        // Verify correct tab on dekstop
+        assert.dom('[data-test-metadata-tab]').isVisible();
+        assert.dom('[data-test-revisions-tab]').isNotVisible();
+        assert.dom('[data-test-tags-tab]').isNotVisible();
 
-    // test for switching views
+        // Verify favicon
+        assert.dom('[data-test-metadata-button]').hasAria('label', 'Close metadata',
+            'Close metadata aria should be present when metadata tab is open.');
+        assert.dom('[data-test-metadata-button] > svg').hasClass('fa-info-circle',
+            'Button should have info circle Font Awesome 5 icon present.');
+
+        // File Metadata
+        assert.dom('[data-test-metadata-header]').hasText('File Metadata',
+            'File metadata header is properly set.');
+        // Title
+        assert.dom('[data-test-file-title-label]').hasText('Title',
+            'File metadata title label text is properly set.');
+        assert.dom('[data-test-file-title]').exists();
+        // Description
+        assert.dom('[data-test-file-description-label]').hasText('Description',
+            'File metadata description label text is properly set.');
+        assert.dom('[data-test-file-description]').exists();
+        // Resource type
+        assert.dom('[data-test-file-resource-type-label]').hasText('Resource type',
+            'File metadata resource type text is properly set.');
+        assert.dom('[data-test-file-resource-type]').exists();
+        // Resource language
+        assert.dom('[data-test-file-language-label]').hasText('Resource language',
+            'File metadata resource language text is properly set.');
+        assert.dom('[data-test-file-language]').exists();
+
+        // Metadata node type
+        const nodeNounElement = document.querySelectorAll('[data-test-metadata-node]')[0].textContent;
+        const nounFound = nodeNoun.some(r => nodeNounElement?.includes(r));
+        assert.equal(nounFound, true, 'Node noun properly set.');
+        // Funder metadata
+        assert.dom('[data-test-target-funder-div]').exists();
+        // Funder name
+        assert.dom('[data-test-target-funder-name-label]').hasText('Funder',
+            'File metadata funder text is properly set.');
+        assert.dom('[data-test-target-funder-name]').exists();
+        // Award title
```

Comment:
Should also verify that the funder name matches what is saved in the node metadata record. Same for the rest of the fields. You can see how to do this in `tests/acceptance/guid-node/metadata-test.ts`.

Reply:
Thank you, this is now updated.